react/Gallery.js

```diff
@@ -90,57 +44,21 @@ class Gallery extends Component {
     )
   }
 
-  updateCache = layoutMode => {
-    this.setState({ prevLayoutMode: layoutMode })
-    this.cache.clearAll()
-  }
-
   render() {
     const {
       products,
       layoutMode,
       runtime: { hints: { mobile } },
-      itemWidth,
     } = this.props
-    // Maps the WindowScroller and the AutoSizer props to a more adequate set of params
-    const mapContainerProps = ({ scroller: { height, scrollTop, isScrolling }, autoSizer: { width } }) => ({ width, height, scrollTop, isScrolling })
 
-    // Updates the cache to recalculate heights if the layoutMode changes
-    if (layoutMode !== this.state.prevLayoutMode) {
-      this.updateCache(layoutMode)
-    }
+    const containerClasses = classNames(`${searchResult.gallery} pa3 bn`, {
+      [searchResult.galleryTwoColumns]: layoutMode === 'small' && mobile,
+    })
 
     return (
       <NoSSR onSSR={this.handleSSR()}>
```

Comment:
I believe the handleSSR is not necessary anymore, since the Server side and the client side are the same, no?